tests/test_qsimh_device.py

```diff
@@ -0,0 +1,400 @@
+# Copyright 2018 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the QSimhDevice class
+"""
+import math
+from unittest.mock import MagicMock, patch
+
+import cirq
+import pennylane as qml
+from pennylane.wires import Wires
+import pytest
+import numpy as np
+
+from pennylane_cirq.qsim_device import QSimhDevice
+
+
+@patch.multiple(QSimhDevice, __abstractmethods__=set())
```

Comment:
Unsure if we need this also here, isn't it just to help with `CirqDevice` which has some abstract methods. Do we still have abstract methods in `QSimhDevice`?

Reply:
Oh, my bad, I copied the tests for `CirqDevice` rather than for `SimulatorDevice`. Thanks for catching this @trbromley! :facepalm: